main.go

```diff
@@ -30,9 +40,26 @@ var (
 )
 
 func main() {
-	connpool := getConnectionPool()
+	var conf Config
+	if _, err := confl.DecodeFile("dbconfig.conf", &conf); err != nil {
```

Comment:
Don't forget to push this config file. And since it contains configs for multiple things, better rename it.

Reply:
I separated config into two files, one for db and one for Kafka and Es, because dbcofig should stay local.